CHANGELOG.md

```diff
@@ -92,6 +92,8 @@ New Features
   point sources distributed using a random walk.  Uses of this profile include
   representing an "irregular" galaxy, or adding this profile to an Exponential
   to represent knots of star formation. (#819)
+- Added support for reading in of unsigned int Images (#715)
```

Comment:
This is a stray addition that already shows up at the top of this section.  (Our ordering scheme is by issue number, so 715 comes before the other items in this list.)